test/mocha/helpers.js

```diff
@@ -32,6 +32,31 @@ exports.createKeystore = async ({
   });
 };
 
+exports.getKeystore = async ({id}) => {
+  const {httpsAgent} = brHttpsAgent;
+  return await KmsClient.getKeystore({id, httpsAgent});
```

Comment:
unnecessary await here and the other new helpers.

The only reason you may see this patter around has to with returning inside try/catch blocks where if you don't await locally, the error is not thrown in the local context and the try/catch does not work as expected.

That's not the case here, so the await is redundant.  Just `return KmsClient.foo();`